### PR TITLE
VxDesign 3.1: Convert ballots to grayscale

### PR DESCRIPTION
**Pull request by jonahkagan**
## Overview

Convert the "all ballots" export and the test deck export to grayscale using `gs` to expedite the printing process.

## Demo Video or Screenshot

Screenshot 2024-02-13 at 9 34 15 AM

## Testing Plan
Manually tested and updated some ballot fixtures

## Checklist

- [ ] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.

- [ ] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: libs/hmpb/render-backend/src/render_ballot.tsx [hunk 1]
```diff
@@ -117,6 +121,33 @@ export function renderDocumentToPdf(document: Document): PDFKit.PDFDocument {
...
+      -sOutputFile=${tmpGrayscalePdfFilePath} \
+      -sDEVICE=pdfwrite \
+      -sColorConversionStrategy=Gray \
+      -dProcessColorModel=/DeviceGray \
+      -dNOPAUSE \
+      -dBATCH \
+      ${tmpPdfFilePath}
+  `);
+  return createReadStream(tmpGrayscalePdfFilePath);
+}
```
[the remark this answers is not in the document]
Comment (jonahkagan) [reply libs/hmpb/render-backend/src/render_ballot.tsx]: nice!

[the remark this answers is not in the document]
Comment (jonahkagan) [reply libs/hmpb/render-backend/src/render_ballot.tsx]: Hmm, I couldn't get this to work well - it was creating bad zip files. I don't think there's a huge cost to the temp files in this case, so I'm going to leave it for now

[the remark this answers is not in the document]
Comment (eventualbuddha) [reply libs/hmpb/render-backend/src/render_ballot.tsx:149]: Hmm. Weird. I tested it locally using `./bin/generate-fixtures`. What were you doing that caused problems? Using the web UI?

[the remark this answers is not in the document]
Comment (jonahkagan) [reply libs/hmpb/render-backend/src/render_ballot.tsx:149]: Yeah, it outputted an invalid zip for "Export All Ballots"

**Pull request merged**